packages/react/src/oidc/vanilla/checkSession.ts

```diff
@@ -21,7 +21,7 @@ export const startCheckSessionAsync = (oidc:any, oidcDatabase:any, configuration
                 return silentLoginAsync({
                     prompt: 'none',
                     id_token_hint: idToken,
-                    scope: 'openid',
+                    scope: configuration.scope,
```

Comment:
thank you very much @cloudlucky ,

it is possible to set :
configuration.scope || 'openid'

Reply:
it has been changed to `configuration.scope || 'openid'`